packages/pg/collection.js

```diff
@@ -61,7 +61,7 @@ const QBProto = Meteor.isClient ?
 QBProto.fetchOne = function fetchOne() {
   const rows = QBProto.fetch.call(this);
   if (rows.length === 0) {
-    throw new Error("fetchOne/fetchValue: query returned no rows");
+    throw new Error(`PG: fetchOne no data`, `fetchOne/fetchValue: query returned no rows.`);
```

Comment:
I think @rclai is correct that we shouldn't error in this case. Returning `undefined` sounds good, because that's not a valid type for data returned from SQL.